setup.py

```diff
@@ -101,11 +101,9 @@
       zip_safe=False,
       install_requires=install_requires,
       extras_require={
-          'testing': testing_extras,
+          'testing': tests_require,
           'docs': docs_extras,
           },
-      tests_require=tests_require,
-      test_suite="pyramid.tests",
```

Comment:
Yeah you can't drop the test_suite and tests_require stuff here. The core isn't interested in dropping `python setup.py test` support just yet.